lib/core/src/Cardano/Wallet/DB/Checkpoints/Model.hs

```diff
@@ -100,35 +120,74 @@ fromGenesis a = Checkpoints $ Map.singleton W.Origin a
 
 -- | Get the checkpoint with the largest 'SlotNo'.
 getLatest :: Checkpoints a -> (W.Slot, a)
-getLatest = from . Map.lookupMax . view #checkpoints 
+getLatest = fromMaybe err . Map.lookupMax . view #checkpoints
   where
-    from = fromMaybe (error "getLatest: there should always be at least a genesis checkpoint")
+    err = error
+        "getLatest: there should always be at least a genesis checkpoint"
 
 -- | Find the nearest 'Checkpoint' that is either at the given point or before.
 findNearestPoint :: Checkpoints a -> W.Slot -> Maybe W.Slot
 findNearestPoint m key = fst <$> Map.lookupLE key (view #checkpoints m)
 
 {-------------------------------------------------------------------------------
-    Delta type for Checkpoints
+        Delta type for Checkpoints
```

Comment:
Hi @paolino 

Is there a reason to make this change?

If not, would it be possible to remove this change, and others like it?

I think it's completely fair game to reformat things if they break our coding standards, but otherwise, I think we should refrain from making changes unless they directly relate to a PR!